README.md

```diff
@@ -15,101 +16,135 @@ Puppet::Syntax is supported with:
 
 For the specific versions that we test against, see the [TravisCI config](.travis.yml).
 
-If you're using `puppetlabs_spec_helper/rake_tasks` and getting unexpected non-zero exit codes, upgrade to [puppetlabs_spec_helper][psh] version 0.8.0 or greater. Versions of `puppetlabs_spec_helper` prior to 0.8.0 conflicted with Puppet::Syntax.
+If you're using `puppetlabs_spec_helper/rake_tasks` and getting unexpected
+non-zero exit codes, upgrade to [puppetlabs_spec_helper][psh] version 0.8.0 or
+greater. Versions of `puppetlabs_spec_helper` prior to 0.8.0 conflicted with
+Puppet::Syntax.
 
 [psh]: https://github.com/puppetlabs/puppetlabs_spec_helper
 
 ## Installation
 
-To install Puppet::Syntax, either add it to your module's Gemfile or install the gem manually.
+To install Puppet::Syntax, either add it to your module's Gemfile or install
+the gem manually.
 
 * To install with the Gemfile, add:
 
-        gem 'puppet-syntax'
+```ruby
+gem 'puppet-syntax'
+```
 
   And then execute:
 
-        $ bundle install
+```sh
+bundle install
+```
 
 * To install the gem yourself, run:
 
-        $ gem install puppet-syntax
+```sh
+gem install puppet-syntax
+```
 
 ## Configuration
 
 To configure Puppet::Syntax, add any of the following settings to your `Rakefile`.
 
 * To exclude certain paths from the syntax checks, set:
 
-        PuppetSyntax.exclude_paths = ["vendor/**/*"]
+```ruby
+PuppetSyntax.exclude_paths = ["vendor/**/*"]
+```
 
 * To use the Puppet 4 ("future") parser in Puppet 3.2 through 3.8, set:
 
-        PuppetSyntax.future_parser = true
+```ruby
+PuppetSyntax.future_parser = true
+```
 
 * To configure specific paths for the Hiera syntax check, specify `hieradata_paths`. This is useful if you use Hiera data inside your module.
 
-        PuppetSyntax.hieradata_paths = ["**/data/**/*.yaml", "hieradata/**/*.yaml", "hiera*.yaml"]
+```ruby
+PuppetSyntax.hieradata_paths = ["**/data/**/*.yaml", "hieradata/**/*.yaml", "hiera*.yaml"]
+```
 
 * To configure specific paths for the Puppet syntax checks or for the templates checks, specify `manifests_paths` or `templates_paths` respectively. This is useful if you want to check specific paths only.
 
-        PuppetSyntax.manifests_paths = ["**/environments/future/*.pp"]
-        PuppetSyntax.templates_paths = ["**/modules/**/templates/*.erb"]
+```ruby
+PuppetSyntax.manifests_paths = ["**/environments/future/*.pp"]
+PuppetSyntax.templates_paths = ["**/modules/**/templates/*.erb"]
```

Comment:
Do we need to add *.epp templates aswell?

Reply:
mmh I don't know. I will leave it as is, I just changed the formatting.